app/gui/src/dashboard/layouts/Drive/Categories/categoriesHooks.tsx

```diff
@@ -185,7 +180,7 @@ export function useLocalCategoryList() {
     rootPath: Path(directory),
     homeDirectoryId: newDirectoryId(Path(directory)),
     label: getFileName(directory),
-    icon: FolderFilledIcon,
+    icon: 'folder_small',
```

Comment:
We need to double-check that it works everywhere because we didn't fully migrate to the new 'Icon' component